resolwe/flow/models/entity.py

```diff
@@ -27,6 +27,9 @@ class Meta(BaseCollection.Meta):
     #: indicate whether `descriptor` is completed (set by user)
     descriptor_completed = models.BooleanField(default=False)
 
+    #: entity type
+    type = models.CharField(max_length=100, null=True, blank=True)
```

Comment:
We probably want index on this field. Add it to serializer and support filtering on the endpoint.